<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowCategories</class>
 <widget class="QWidget" name="MainWindowCategories">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Categories</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="btnRefresh">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrArea">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrAreaContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>620</width>
        <height>328</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="grpFavourite">
         <property name="title">
          <string>Favourite</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grpAll">
         <property name="title">
          <string>Top</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
